Src/Microsoft.Dynamic/Microsoft.Dynamic.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project>
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is there a reason that this changed? Did you find a problem with the current implementation?